coord/src/dist_aggregate.c

```diff
@@ -82,8 +82,8 @@ static int netCursorCallback(MRIteratorCallbackCtx *ctx, MRReply *rep) {
       RedisModule_Log(NULL, "warning", "Error returned for CURSOR.DEL command from shard");
     }
     // Discard the response, and return REDIS_OK
-    MRReply_Free(rep);
     MRIteratorCallback_Done(ctx, MRReply_Type(rep) == MR_REPLY_ERROR);
```

Comment:
Is this a use after free? How come our sanitizer did not catch it?

Reply:
Good question. It does now :)

Reply:
Do you mean it suddenly started to catch it? We should try to understand why it was missed.

Reply:
It wasn't covered before. This was a pretty resent change.
Now that we delete pending cursors when we delete a user cursor, and with the inhancement to the tests, we now cover it